kbfsgit/file_info.go

```diff
@@ -7,41 +7,63 @@ package kbfsgit
 import (
 	"os"
 	"time"
+
+	"github.com/keybase/kbfs/libfs"
+	"github.com/keybase/kbfs/libkbfs"
 )
 
 // FileInfo is a wrapper around libkbfs.EntryInfo that implements the
 // os.FileInfo interface.
 type FileInfo struct {
+	fs       *FS
+	ei       libkbfs.EntryInfo
+	fullpath string
 }
 
 var _ os.FileInfo = (*FileInfo)(nil)
 
 // Name implements the os.FileInfo interface for FileInfo.
 func (fi *FileInfo) Name() string {
-	return ""
+	return fi.fullpath
```

Comment:
The billy comments are pretty clear that it should be the path that's passed to `Open`, which in billy-land is the full path: https://github.com/src-d/go-billy/blob/d0dc16de10d86398cc9aba4513a2453b401eceb3/fs.go#L131